README.md

```diff
@@ -71,3 +74,4 @@ If you have any cool content, please link to it via a PR.
 + Tutorials
   + [LiveOverflow](https://www.youtube.com/channel/UClcE-kVhqyiHCcjYwcpfj9w)
   + [JackkTutorials](https://www.youtube.com/user/JackkTutorials)
+  + [GynvaelEN](https://www.youtube.com/watch?v=ZSlJVK8DBiY)
```

Comment:
Maybe better to link to the channel (https://www.youtube.com/channel/UCCkVMojdBWS-JtH7TliWkVg) itself rather than a specific live stream